k9mail/src/main/java/com/fsck/k9/activity/setup/AccountSetupBasics.java

```diff
@@ -139,6 +152,10 @@ protected void onRestoreInstanceState(Bundle savedInstanceState) {
         showPassword(mShowPasswordCheckBox.isChecked());
     }
 
+    /**
+     * call before the view is created
+     * @param savedInstanceState
+     */
     @Override
```

Comment:
Some of this Javadoc is either Android Activity lifecycle basics or obvious from the the message name. Code comments should really be limited to explaining where we are doing something odd. If you can get it from the API docs or (preferrably) the name of the method itself, we don't need a comment - which can easily become inaccurate following refactoring.